nightly-benchmark/run-dgx-benchmark.sh

```diff
@@ -30,6 +30,7 @@ popd
 srun -N1 python nightly-run.py > dgx_raw_data.txt
 echo "Copy sitecustomize.py..."
 cp sitecustomize.py ${CONDA_PREFIX}/lib/python3.8/sitecustomize.py
+export DASK__OPTIMIZATION__FUSE__ACTIVE=False
```

Comment:
Looks like this is setting "_optimization.fuse.active"

```suggestion
export DASK_OPTIMIZATION__FUSE__ACTIVE=False
```

Reply:
Thanks!  Good catch!